pom.xml

```diff
@@ -927,6 +952,79 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <properties>
+            <property>
+              <name>listener</name>
+              <value>org.jboss.pnc.test.listener.PrintTestNameListener</value>
+            </property>
+          </properties>
+        </configuration>
+        <executions>
+          <execution>
```

Comment:
Each tests category is executed in separate phase - this gives us flexibility (e.g. run unit tests with fork count 2C; remote and container tests without forks).